A one-line fix from upstream to fix a byte-compilation issue in Python 3.

--- requests-1.2.3/requests/packages/urllib3/contrib/ntlmpool.py.orig~	2013-04-30 10:39:29.000000000 -0700
+++ requests-1.2.3/requests/packages/urllib3/contrib/ntlmpool.py	2014-04-17 08:33:25.770194371 -0700
@@ -33,7 +33,7 @@
     def __init__(self, user, pw, authurl, *args, **kwargs):
         """
         authurl is a random URL on the server that is protected by NTLM.
-        user is the Windows user, probably in the DOMAIN\username format.
+        user is the Windows user, probably in the DOMAIN\\username format.
         pw is the password for the user.
         """
         super(NTLMConnectionPool, self).__init__(*args, **kwargs)
